cmd/kubeadm/app/apis/kubeadm/v1beta2/doc.go

```diff
@@ -233,12 +233,12 @@ limitations under the License.
 // 	scheduler:
 // 	  extraArgs:
 // 	    address: "10.100.0.1"
-// 	extraVolumes:
-// 	- name: "some-volume"
-// 	  hostPath: "/etc/some-path"
-// 	  mountPath: "/etc/some-pod-path"
-// 	  readOnly: false
-// 	  pathType: File
+// 	  extraVolumes:
```

Comment:
please fix this in v1beta1 too:
https://github.com/kubernetes/kubernetes/blob/28d559fcc23de22fe242093310cd79a58abd6f90/cmd/kubeadm/app/apis/kubeadm/v1beta1/doc.go#L241

Reply:
done.